lib/alephant/broker.rb

```diff
@@ -10,42 +10,11 @@ def self.handle(load_strategy, env)
     end
 
     def self.config
-      @@configuration
+      @configuration
     end
 
     def self.config=(c)
-      @@configuration = c
-    end
-
-    class Application
-      attr_reader :load_strategy
-
-      def initialize(load_strategy, c = nil)
-        Broker.config = c unless c.nil?
-        @load_strategy = load_strategy
-      end
-
-      def call(env)
-        send response_for(environment_for(env))
-      end
-
-      def environment_for(env)
-        Environment.new env
-      end
-
-      def response_for(call_environment)
-        Broker.handle(load_strategy, call_environment)
-      end
-
-      def send(response)
-        [
-          response.status,
-          response.headers,
-          [
-            response.content.to_s
-          ]
-        ]
-      end
+      @configuration = c
```

Comment:
yup, as it's a module there is only one "instance" of it, so no need to the `@@` variable.